samples/maven/.gitattributes

```diff
@@ -1,6 +0,0 @@
-#
```

Comment:
Can you please keep these files in the samples and copy it to the root.
The samples are independent projects where the idea is that you as a user can copy this as a starting point of a new project.